### PR TITLE
Add Nevermore Activated Carbon Filter

### PR DESCRIPTION
**Pull request by eduncan911**
* [X] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [X] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [X] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [X] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [X] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [X] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [X] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [X] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [X] Submitted STL files are oriented and scaled properly for printing.
  * [X] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?

Nevermore Micro V4 and V5 Duo !!

#### How was it tested? 

Dozens of people have printed and actively using it, and many variants.

#### Any background context you want to provide?

It's the Nevermore Micro !!  :)

I have been working with the original Nevermore author, @0ndsk4, to clean up the original repository, add a bunch of information, some icky images, create clean and precise BOMs, and get this published.

With his direct permission, I am publishing all of his work for him, from his new Github Org.

The original repository is being deprecated in favor of this new repository we are now publishing mods from, https://github.com/nevermore3d, including the VoronUsers branch located there.

#### Screenshots (if appropriate)

image

image

image

#### Further notes

It's the Nevermore !!  :)  Finally, merged into the VoronUsers group!

### CODE REVIEW

**Comment by eduncan911** [comment]
Removed the 1 space.  This PR is ready for review.

**Comment by eduncan911** [comment]
# DO NOT MERGE

We're going to greatly reduce the README as it's constantly being updated, and we do not want to keep open PR as we build it out.

In the next few days, we should have the new README condensed and ready to go.  This comment will be removed, and a new one added stating it is ready for review.

File: printer_mods/0ndsk4/Nevermore_Micro/README.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+  - 2 for seating plenum lid to plenum
+- 8x 6x3mm cylindrical magnets
+- 1x 2 pin JST header
+- 4x M3x16 BHCS
+  - for heat inserts that go into the four fan tabs
+- 1x M3x18 BHCS
+  - ONLY for delta and v1.8 variants
+- 1x M3x6 BHCS
+- 1x M3x4 BHCS
+  - for the extra hole in the plenum@lid for symmetry (it doesn’t attach to anything and is totally optional)
```
Comment (bdbell) [inline printer_mods/0ndsk4/Nevermore_Micro/README.md:75]: The random @ looks like a typo.

File: printer_mods/0ndsk4/Nevermore_Micro/V5_Duo/V2/Readme.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Four parts; Plenum+Lid, and Cartridge+Lid.
+
+Choose between regular and XL cartridge. XL is just 10mm longer, which can make a big difference in air flow.
+
+Choose XL for max carbon content and filter duration, and if you dont mind the fan (200Pa+) has to run faster/noisier. You probably have to use pellets instead of granular carbon too.
+
+Choose regular if you have granular carbon and/or a wekaer fan, and/or wish to have a more silent experience. Regular also allows for somewhat higher airflow, which could make a small difference compared to XL in chamber heat up times.
```
Comment (bdbell) [inline printer_mods/0ndsk4/Nevermore_Micro/V5_Duo/V2/Readme.md:7]: 'weaker' is misspelled.

**Comment by bdbell** [comment]
There appears to be an error in the STL or object V5_Duo/1.8/V18_cartridge_RC2.stl:
image

**Comment by eduncan911** [comment]
@0ndsk4 have you updated these files in the main repo?  if so, I can copy them over.

**Comment by 0ndsk4** [comment]
> There appears to be an error in the STL or object V5_Duo/1.8/V18_cartridge_RC2.stl: image
> 
> Also found on V5_Duo/V2/Duo_Cartridge.stl and V5_Duo/V2/Duo_XL_Cartridge.stl

That’s no error as far as I can see. It’s built in support, supposed to be removed after the print.

**Comment by 0ndsk4** [comment]
> There appears to be an error in the STL or object V5_Duo/1.8/V18_cartridge_RC2.stl: image
> 
> Also found on V5_Duo/V2/Duo_Cartridge.stl and V5_Duo/V2/Duo_XL_Cartridge.stl

That’s no error as far as I can see. It’s built in support, supposed to be removed after the print.

**Comment by newphreak** [comment]
@eduncan911 can you update it with the latest touches from the [Nevermore_Micro](https://github.com/nevermore3d/Nevermore_Micro) repo?

**Comment by eduncan911** [comment]
@newphreak rebased and all set!

**Comment by FHeilmann** [comment]
Hi, 

I want to thank you for your submission and want to personally apologize that your submission has not gotten the review it deserves.
We are in the process to restructure the repo to streamline reviews and hopefully solve this issue in the future.

Unfortunately, this means that we will require you to resubmit the mod under the new system.

Please check [here](https://github.com/VoronDesign/VoronUsers/wiki/VoronUsers-Overhaul) for details and instructions on how to proceed. 
If you have any questions, please reach out in the #voronuser_mods channel on the VoronDesign discord.
If you have since moved on from this mod and don't want to resubmit, no further action is necessary from your side.

Bests,

Florian a.k.a. Haribro

**Pull request closed without merging**